app/assets/javascripts/app/collections/comments.js

```diff
@@ -11,7 +11,7 @@ app.collections.Comments = Backbone.Collection.extend({
   make : function(text){
     var self = this;
 
-    var comment = new app.models.Comment({text: text });
+    var comment = new app.models.Comment({ 'text': text });
```

Comment:
The other strings of the file use simple-quote. Do you have to change all the strings of this file?